src/Sylius/Bundle/UiBundle/Resources/views/Grid/_default.html.twig

```diff
@@ -1,77 +1 @@
-{% set criteria = app.request.query.get('criteria') %}
-
-{% if definition.enabledFilters|length > 0 %}
-    <div class="ui hidden divider"></div>
-    <div class="ui styled fluid accordion">
-        <div class="title {% if criteria is not null %}active{% endif %}">
-            <i class="dropdown icon"></i>
-            <i class="filter icon"></i>
-            {{ 'sylius.ui.filters'|trans }}
-        </div>
-        <div class="content {% if criteria is not null %}active{% endif %}">
-            <form method="get" action="{{ path }}" class="ui loadable form" novalidate>
-                <div class="sylius-filters">
-                    {% for filter in definition.enabledFilters|filter(filter => filter.enabled)|sort_by('position') %}
-                        <div class="sylius-filters__field">
-                            {{ sylius_grid_render_filter(grid, filter) }}
-                        </div>
-                    {% endfor %}
-                </div>
-                {{ buttons.filter() }}
-                {{ buttons.resetFilters(path) }}
-            </form>
-        </div>
-    </div>
-{% endif %}
-
-<div class="ui hidden divider"></div>
-<div class="sylius-grid-wrapper">
-    <div class="sylius-grid-nav">
-        {% if data|length > 0 and definition.actionGroups.bulk is defined and definition.getEnabledActions('bulk')|length > 0 %}
-            <div class="sylius-grid-nav__bulk">
-                {% for action in definition.getEnabledActions('bulk') %}
-                    {{ sylius_grid_render_bulk_action(grid, action, null) }}
-                {% endfor %}
-            </div>
-        {% endif %}
-        <div class="sylius-grid-nav__pagination">
-            {{ pagination.simple(data) }}
-        </div>
-        {% if definition.limits|length > 1 and data|length > min(definition.limits) %}
-            <div class="sylius-grid-nav__perpage">
-                <div class="ui fluid one menu sylius-paginate">
-                    {{ pagination.perPage(data, definition.limits) }}
-                </div>
-            </div>
-        {% endif %}
-    </div>
-
-    {% if data|length > 0 %}
-        <div class="ui segment spaceless sylius-grid-table-wrapper">
-            <table class="ui sortable stackable very basic celled table" {{ sylius_test_html_attribute('grid-table') }}>
-                <thead>
-                <tr>
-                    {{ table.headers(grid, definition, app.request.attributes) }}
-                </tr>
-                </thead>
-                <tbody {{ sylius_test_html_attribute('grid-table-body') }}>
-                {% for row in data %}
-                    {{ table.row(grid, definition, row) }}
-                {% endfor %}
-                </tbody>
-            </table>
-        </div>
-    {% else %}
-        {{ messages.info('sylius.ui.no_results_to_display') }}
-    {% endif %}
-    {{ pagination.simple(data) }}
-</div>
+{{ sylius_template_event('sylius.admin.grid.content', _context) }}
```

Comment:
We could think about more then one event being triggered here, like generic grid, admin/shop and resource if exists. You may check how it may be done here for example here: [src/Sylius/Bundle/AdminBundle/Resources/views/Crud/create.html.twig](https://github.com/Sylius/Sylius/blob/3efc44a3591f5be998e38c905d36bb0eac89bfee/src/Sylius/Bundle/AdminBundle/Resources/views/Crud/create.html.twig#L23)